specs/ledger/hs/src/Ledger/Core.hs

```diff
@@ -83,17 +83,17 @@ verify (VKey vk) vd (Sig sd sk) = vk == sk && vd == sd
 -- Slots and Epochs
 ---------------------------------------------------------------------------------
 
-newtype Epoch = Epoch Natural
+newtype Epoch = Epoch Word64
   deriving (Eq, Ord, Show, HasTypeReps)
 
-newtype Slot = Slot Natural
+newtype Slot = Slot { unSlot :: Word64 }
```

Comment:
what is the reason for the `unSlot` field name here?

Reply:
That seems to be the convention, as I learn from @ruhatch, which seems to be part of our (draft) [new best practices](https://festive-hoover-f1dec6.netlify.com/haskell/best-practice/).

Reply:
ok, didn't know that, fine with me, but then why not below for `SlotCount` aka `Duration`?

Reply:
Oh, just because we didn't need to use an accessor so far.